src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/open_deadline_cloud_dialog.py

```diff
@@ -52,13 +52,38 @@ def create_deadline_dialog(parent=None) -> SubmitJobToDeadlineDialog:
 
     # For the output path, first check for a value in the Scene settings
     if os.path.dirname(bpy.context.scene.render.filepath):
-        settings.output_path = os.path.dirname(bpy.context.scene.render.filepath)
+        settings.output_directory = os.path.dirname(bpy.context.scene.render.filepath)
     # If none, use the one in Preferences
     elif bpy.context.preferences.filepaths.render_output_directory:
-        settings.output_path = bpy.context.preferences.filepaths.render_output_directory
+        settings.output_directory = bpy.context.preferences.filepaths.render_output_directory
     # If neither of these are set, use the job bundle directory by default
     else:
-        settings.output_path = os.path.dirname(bpy.context.blend_data.filepath)
+        settings.output_directory = os.path.dirname(bpy.context.blend_data.filepath)
+
+    if bpy.path.basename(bpy.context.scene.render.filepath):
+        settings.output_file_prefix = bpy.path.basename(bpy.context.scene.render.filepath)
+
+    # When setting the output format, we can only access the extension from the Blender API.
+    # Here, we map them to formats available in the submitter GUI combobox.
+    file_formats = {
```

Comment:
Could a mapping be created like was done previously for KeyShot where there was a [similar issue](https://github.com/aws-deadline/deadline-cloud-for-keyshot/pull/3/files)? That way the definitions of the mappings only live in one place.